.github/workflows/deploy_terraform.yml

```diff
@@ -93,40 +121,46 @@ jobs:
 
   run_deploy:
     name: Run Deploy - ${{ needs.pre_job.outputs.env_name }}
-    concurrency:
+    concurrency: 
       group: ${{ github.workflow }}-${{ needs.pre_job.outputs.env_name }}
       cancel-in-progress: true
     needs:
       - pre_job
       - approve_deploy
-    if: needs.confirm_changes.outputs.change_count > '0'
+    if: needs.confirm_changes.outputs.change_count != '0'
     runs-on: ubuntu-latest
     environment: ${{ needs.pre_job.outputs.env_name }}
     defaults:
       run:
         working-directory: operations/app/terraform/vars/${{ needs.pre_job.outputs.env_name }}
     steps:
       - name: Check Out Changes
-        uses: actions/checkout@d632683dd7b4114ad314bca15554477dd762a938
+        uses: actions/checkout@v4
       - name: Connect to VPN and login to Azure
         uses: ./.github/actions/vpn-azure
         with:
           env-name: ${{ needs.pre_job.outputs.env_name }}
           tls-key: ${{ secrets.TLS_KEY }}
-          ca-cert: ${{ secrets.CA_CRT}}
+          ca-cert: ${{ secrets.CA_CRT }}
           user-crt: ${{ secrets.USER_CRT }}
           user-key: ${{ secrets.USER_KEY }}
           sp-creds: ${{ env.AZURE_CREDENTIALS }}
           tf-auth: true
       - name: Use specific version of Terraform
-        uses: hashicorp/setup-terraform@b9cd54a3c349d3f38e8881555d616ced269862dd
+        uses: hashicorp/setup-terraform@v1
         with:
-          terraform_version: 1.7.4
-          terraform_wrapper: false
-      - name: Run Terraform
+          terraform_version: 1.7.4  
```

Comment:
Let's set [terraform_wrapper](https://github.com/hashicorp/setup-terraform) to false